theme/less/_homepage.less

```diff
@@ -0,0 +1,82 @@
+body div
+{
+  grid-template-areas: 
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+main article
+{
+  display: grid;
+  grid-template-columns: auto auto;
+  column-gap: 20px;
+}
+
+main article > *:first-child
+{
+  grid-column: 1 e("/") 3;
+}
+
+.box
+{
+  position: relative;
+  background-color: #f2f2f2;
+  border-radius: 8px;
+  border: 1px solid #dcdcdc;
+  padding: 10px 10px 60px 10px;
+  margin: 20px 0 0 0;
+  z-index: -1;
+}
+
+.box img 
+{
+  float: left;
+  padding: 0 15px 0 0;
+}
+
+.box h3 
+{
+  margin: 0;
+}
+
+.box a
+{
+  position: absolute;
+  right: 20px;
+  bottom: 20px;
+  text-decoration: none;
+  background: linear-gradient(#ffffff, #ececec);
+  border: 1px solid #dbdbdb;
+  border-radius: 6px;
+  padding: 10px;
+}
+
+.box a, .box a:hover, .box a:active, .box a:visited
+{
+  color: inherit;
+}
+
+.box a:hover
+{
+  cursor: pointer;
+  background: linear-gradient(#d1d1d1, #fcfcfc);
+}
+
+.box a:active
+{
+  box-shadow: inset 0 0 8px 4px #d4d4d4;
+}
+
+@media screen and (max-width: 770px) 
+{
+  main article
+  {
+    grid-template-columns: auto;
+  }
+
+  main article > :first-child
```

Comment:
Detail: This selector is inconsistent with the parent one and still thinking maybe we can use a declarative naming here, as it's about page's content manipulation. `.merge-cells`?

Reply:
I don't think that here we exactly are merging cell, because the property `grid-column: 1` (or grid-column: 1 / 2) means there's only one column.

Reply:
Okey, agree leaving as it is right now.